src/org/gk/render/RenderableSwitchHelper.java

```diff
@@ -110,12 +113,14 @@ private void copyDispayProperties(Renderable source, Renderable target) {
 	
 	/**
 	 * Switch a ReactionNode to a RenderablePathway. An empty reaction can be switched to pathway
-	 * with the same display name and properties. However, a non-empty reaction cannot switch to
-	 * pathway, a IllegaleArgument exception will be thrown.
-	 * @param node
-	 * @return
+	 * with the same display name and properties. However, a non-empty reaction cannot switch to a
+	 * pathway - an IllegalArgumentException will be thrown.
+	 * 
+	 * @param node ReactionNode objects to switch to a RenderablePathway object
+	 * @return RenderablePathway object created from the ReactionNode passed
+	 * @throws IllegalArgumentException Thrown if the ReactionNode object contains Node objects
 	 */
-	public RenderablePathway switchReactionToPathway(ReactionNode node) {
+	public RenderablePathway switchReactionToPathway(ReactionNode node) throws IllegalArgumentException {
```

Comment:
Why didn't this thrown an exception before? Why add it?

Reply:
It wasn't included in the method header before, but the body does have a statement throwing the exception.  It wasn't forced to be included because it is a runtime/unchecked exception rather than a checked one, but I believe there are other places in the codebase which document IllegalArgumentExceptions, so I included it for consistency.

Reply:
Ok.